README.md

```diff
@@ -97,7 +97,14 @@ Record variants are represented in the same way as if the nested structure was d
 ["X",{"v":0}]
 ```
 
-Record variants are currently not supported for extensible variant types.
+Alternative representations of variants can be chosen using the option `variants` (i.e. ``[@@deriving yojson { variants = `Adjacent ("tag", "contents") }]``). The following table lists the ones that are supported:
+
+| `type _ =`                | Value            | ``variants = `Array``  | ``variants = `External`` | ``variants = `Internal "type"`` | ``variants = `Adjacent ("tag", "contents")`` | ``variants = `Native`` |
+| ------------------------- | ---------------- | ---------------------- | ------------------------ | ------------------------------- | -------------------------------------------- | ---------------------- |
+| `\| RA`                   | `RA`             | `["RA"]`               | `"RA"`                   | `{"type": "RA"}`                | `{"tag": "RA"}`                              | `<"RA">`               |
+| `\| RB of int`            | `RB 42`          | `["RB", 42]`           | `{"RB": 42}`             | (not supported)                 | `{"tag": "RB", "contents": 42}`              | `<"RB":42>`            |
+| `\| RC of int * string`   | `RC (42, "foo")` | `["RC", 42, "foo"]`    | `{"RC": [42, "foo"]}`    | (not supported)                 | `{"tag": "RC", "contents": [42, "foo"]}`     | `<"RC":[42, "foo"]>`   |
+| `\| RD of { z : string }` | `RD {z = "foo"}` | `["RD", {"z": "foo"}]` | `{"RD": {"z": "foo"}}`   | `{"type": "RD", "z": "foo"}`    | `{"tag": "RD", "contents": {"z": "foo"}}`    | `<"RD":{"z": "foo"}>`  |
```

Comment:
The table of examples is nice, but I would like to also have a list of all supported choices, with a description of the layout in English prose. (Before or after.)

Reply:
Done.